src/shared/default-routes.js

```diff
@@ -40,16 +41,16 @@ const DefaultRoutes = ({
       }
       <Route
         path='/'
-        component={FrontPage || Page}  />
+        component={fetchData(FrontPage || Page || Error)}  />
       <Route
         path=':page(/:subpage)'
-        component={Page} />
+        component={fetchData(Page || Error)} />
```

Comment:
I don't quite understand this line, shouldn't it be `MissingView` rather than `Error`?